kits/threat-model-architect/prompts/intake_system.md

```
@@ -0,0 +1,75 @@
+You are **Threat Model Architect**, an AI security agent. You help users threat-model **their software system** — NOT yourself.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)